File: CArticulo/CArticulo/Articulo.cs
using System;
namespace CArticulo
{
	public class Articulo
	{
		public Articulo()
		{
			nombre = "";
		}

		private long id;
		private string nombre;
		private long categoriaId;
		private decimal precio;

		public long Id
		{
			get { return id; }
			set { id = value; }
		}

		public string Nombre
		{
			get { return nombre; }
			set { nombre = value; }
		}

		public long CategoriaId
		{
			get { return categoriaId; }
			set { categoriaId = value; }
		}

		public decimal Precio
		{
			get { return precio; }
			set { precio = value; }
		}
	}
}

File: GVenta/src/main/java/serpis/ad/Articulo.java
package serpis.ad;

import java.math.BigDecimal;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Entity
public class Articulo {
	@Id
	@GeneratedValue(strategy=GenerationType.IDENTITY)
	private long id;
	private String nombre;
	private BigDecimal precio;
	private long categoria;
	
	public void setId(long id) {
		this.id = id;
	}
	
	public long getId() {
		return id;
	}
	
	public String getNombre() {
		return nombre;
	}
	
	public void setNombre(String nombre) {
		this.nombre = nombre;
	}
	
	public BigDecimal getPrecio() {
		return precio;
	}
	
	public void setPrecio(BigDecimal precio) {
		this.precio = precio;
	}
	
	public long getCategoria() {
		return categoria;
	}
	
	public void setCategoria(long categoria) {
		this.categoria = categoria;
	}
	
	@Override
	public String toString() {
		return String.format("[%s] %s %s€ %s", id, nombre, precio, categoria);
	
	}
}